### PR TITLE
Replace Landmark messages with StationaryObject messages

### PR DESCRIPTION
**Pull request by carsten-kuebler**
A second suggestion for issue #166. (and revert PR #166).

Use StationaryObject as Landmark object (in GT and for sensor detections).

### CODE REVIEW

File: osi_detectedobject.proto [hunk 1]
```diff
@@ -382,3 +382,82 @@ enum MeasurementState
...
+    // A list of sensors which detected this detected entity.
+    //
+    // If \c SensorData has detected entities and all detections are missing, 
+    // then e.g. the number of sensors can confirm the #existence_probability.
+    // 
+    // \note This information can be determined via the detected entities'
+    // detections ( \c ...Detection::object_id = 'this detected entity' ) and
+    // the sensors (their IDs) to which these detections belong.
+    //
+    repeated Identifier sensor_id_list = 6;
```
Comment (CarloVanDriestenBMW) [inline osi_detectedobject.proto]: remove _list

File: osi_detectedobject.proto [hunk 2]
```diff
@@ -382,3 +382,82 @@ enum MeasurementState
...
+    // The description of the stationary object (e.g. landmark).
+    //
+    optional StationaryObject stationary_object = 1;
+
+    // The estimated probability that this candidate is the true value.
+    // The sum of all \c #candidate_probability must be one.
+    //
+    // Range: [0,1]
+    //
+    optional double candidate_probability = 2;
```
Comment (CarloVanDriestenBMW) [inline osi_detectedobject.proto]: remove candidate_

**Comment by CarloVanDriestenBMW** [comment]
@carsten-kuebler I will push the changes and afterwards we discuss how to cleverly integrate the classes of vehicles in GT into the classProbability - I think we will do a new message with: 
`new_message { vehicle.type; double probability}`

**Comment by carsten-kuebler** [comment]
@CarloVanDriestenBMW looks good...

**Pull request merged**